src/define.js

```diff
@@ -688,7 +690,16 @@ make = {
 					return setter;
 				} else {
 					return function setter(newValue){
-						return set.call(this, canReflect.convert(newValue, type));
+						try {
```

Comment:
we should avoid this extra `try/catch` in production cc @phillipskevin

Reply:
+1, please make this part dev-only with steal-remove-start/stop comments @cherifGsoul

Reply:
Updated!
cc @matthewp @justinbmeyer